ui/Rhino/AGS/dev/AGS_toogle_autoupdate_cmd.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import scriptcontext as sc
+
+import compas_rhino
+
+
+__commandname__ = "AGS_toggle_autoupdate"
+
+
+def RunCommand(is_interactive):
```

Comment:
as with the maxdev, i think this should be part of the general settings dialog...

Reply:
This was again created to compatibility with mac